### PR TITLE
Feature/add tests

### PR DESCRIPTION
**Pull request by mateuszklimek**

### CODE REVIEW

File: macros/public/store/save_results_history.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                {% endif %}
+
+                {% do to_insert.append({ 'table_name': model_name, 'column_name': el.node.column_name , 'test_name': el.node.name, 'status': el.status.name}) %}
+            {% endif %}
+        {% endfor %}
+
+        {% set run_started_at_str = run_started_at.strftime('%Y-%m-%d %H:%M:%S') %}
+
+        {% set insert_query %}
+            insert into {{ ref('re_data_test_history')}} (table_name, column_name, test_name, status, runned_at) values
```
Comment (dejii) [inline macros/public/store/save_results_history.sql:30]: i think we should change `runned_at` to `run_at`.

Comment (mateuszklimek) [reply macros/public/store/save_results_history.sql]: updated ;)

File: macros/public/store/save_results_history.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    -- depends_on: {{ ref('re_data_test_history') }}
+
+    {% set re = modules.re %}
+
+    {% if execute and results %}
+        {% set to_insert = [] %}
+        {% for el in results %}
+            {% if el.node.resource_type.name == 'Test' %}
+                {% set any_refs = re.findall("ref\(\'(?P<name>.*)\'\)", el.node.test_metadata.kwargs['model']) %}
+                {% set any_source = re.findall("source\(\'(?P<one>.*)\'\,\s+\'(?P<two>.*)\'\)", el.node.test_metadata.kwargs['model']) %}
```
Comment (dejii) [inline macros/public/store/save_results_history.sql:12]: i'm wondering if `depends_on` can help here. The node contains information about its ancestors(macros/seeds/models) and it seems to me that's the thing extracted here. right?

Comment (mateuszklimek) [reply macros/public/store/save_results_history.sql:12]: yeah, I was considering to use depeds_on but concern was it's more then one thing there, so we would need to make some assumption about the order they appear etc. Maybe we could take first or the last one..

**Pull request merged**